quartz/components/ExplorerNode.tsx

```diff
@@ -0,0 +1,197 @@
+// @ts-ignore
+import { Data } from "vfile"
```

Comment:
prefer `QuartzPluginData` to `Data` just for semantics

Reply:
wait I meant that we should use `QuartzPluginData` instead of `Data` haha lol

Reply:
ahh my bad, was late already 😅
fixed now